x-pack/legacy/plugins/apm/public/components/app/TransactionOverview/__jest__/TransactionOverview.test.tsx

```diff
@@ -22,12 +22,23 @@ import * as useServiceTransactionTypesHook from '../../../../hooks/useServiceTra
 import { fromQuery } from '../../../shared/Links/url_helpers';
 import { Router } from 'react-router-dom';
 import { UrlParamsProvider } from '../../../../context/UrlParamsContext';
+import { KibanaCoreContext } from '../../../../../../observability/public';
+import { LegacyCoreStart } from 'kibana/public';
 
 jest.spyOn(history, 'push');
 jest.spyOn(history, 'replace');
 
 jest.mock('ui/kfetch');
 
+const coreMock = ({
+  // http: {
+  //   basePath: {
+  //     prepend: (path: string) => `/basepath${path}`
+  //   }
+  // },
```

Comment:
Maybe you can delete these commented lines?

Reply:
Fixed in b9d014d752b.